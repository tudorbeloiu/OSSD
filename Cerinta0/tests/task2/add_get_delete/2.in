40
1
3
239
1360
201
2409
1
3578
1
3
2
6488
246
8946
8
2237
3
246
2
2
1
4
196
1084
31
7863
42
6378
207
2738
1
6
254
406
25
819
148
1729
165
3119
35
4600
200
6444
2
200
2
8
1
1
20
7125
2
196
2
31
1
6
13
4084
145
2402
171
392
16
7503
212
7955
57
1973
3
171
3
148
1
8
127
7641
111
5312
132
2695
98
2832
230
1474
70
5042
109
6062
178
1223
3
230
1
3
19
5212
172
2001
205
1677
2
207
3
42
2
70
3
70
2
16
3
2
2
35
2
127
3
239
2
212
1
3
112
4269
45
8143
202
4593
2
172
2
98
1
2
161
1411
58
53
3
212
2
111
1
3
14
6082
105
7045
3
6315
1
4
245
1579
255
8682
135
4484
155
6997
1
2
151
4916
226
2100
1
10
186
4416
193
5508
95
988
241
1373
30
8811
160
8134
93
6991
199
7736
238
2387
34
3729
1
8
27
430
216
1339
153
1122
17
3352
144
2271
89
6193
208
5780
125
5722
1
10
167
7422
48
1205
39
2162
209
3556
156
8347
212
7007
222
1428
166
720
163
412
36
4247
1
6
85
5852
2
7402
217
8005
91
4892
24
4621
228
1053
