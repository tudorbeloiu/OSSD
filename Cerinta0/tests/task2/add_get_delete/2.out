239: ((0, 0), (0, 169))
201: ((0, 170), (0, 471))
1: ((0, 472), (0, 919))
2: ((1, 0), (1, 810))
246: ((0, 0), (0, 0))
8: ((2, 0), (2, 279))
239: ((0, 0), (0, 169))
201: ((0, 170), (0, 471))
1: ((0, 472), (0, 919))
2: ((1, 0), (1, 810))
8: ((2, 0), (2, 279))
((1, 0), (1, 810))
196: ((1, 811), (1, 946))
31: ((3, 0), (3, 982))
42: ((4, 0), (4, 797))
207: ((2, 280), (2, 622))
254: ((0, 920), (0, 970))
25: ((2, 623), (2, 725))
148: ((2, 726), (2, 942))
165: ((5, 0), (5, 389))
35: ((5, 390), (5, 964))
200: ((6, 0), (6, 805))
((6, 0), (6, 805))
((2, 0), (2, 279))
20: ((7, 0), (7, 890))
((1, 811), (1, 946))
((3, 0), (3, 982))
13: ((8, 0), (8, 510))
145: ((8, 511), (8, 811))
171: ((0, 971), (0, 1019))
16: ((9, 0), (9, 937))
212: ((10, 0), (10, 994))
57: ((11, 0), (11, 246))
239: ((0, 0), (0, 169))
201: ((0, 170), (0, 471))
1: ((0, 472), (0, 919))
254: ((0, 920), (0, 970))
2: ((1, 0), (1, 810))
196: ((1, 811), (1, 946))
8: ((2, 0), (2, 279))
207: ((2, 280), (2, 622))
25: ((2, 623), (2, 725))
148: ((2, 726), (2, 942))
31: ((3, 0), (3, 982))
42: ((4, 0), (4, 797))
165: ((5, 0), (5, 389))
35: ((5, 390), (5, 964))
200: ((6, 0), (6, 805))
20: ((7, 0), (7, 890))
13: ((8, 0), (8, 510))
145: ((8, 511), (8, 811))
16: ((9, 0), (9, 937))
212: ((10, 0), (10, 994))
57: ((11, 0), (11, 246))
239: ((0, 0), (0, 169))
201: ((0, 170), (0, 471))
1: ((0, 472), (0, 919))
254: ((0, 920), (0, 970))
2: ((1, 0), (1, 810))
196: ((1, 811), (1, 946))
8: ((2, 0), (2, 279))
207: ((2, 280), (2, 622))
25: ((2, 623), (2, 725))
31: ((3, 0), (3, 982))
42: ((4, 0), (4, 797))
165: ((5, 0), (5, 389))
35: ((5, 390), (5, 964))
200: ((6, 0), (6, 805))
20: ((7, 0), (7, 890))
13: ((8, 0), (8, 510))
145: ((8, 511), (8, 811))
16: ((9, 0), (9, 937))
212: ((10, 0), (10, 994))
57: ((11, 0), (11, 246))
127: ((12, 0), (12, 955))
111: ((11, 247), (11, 910))
132: ((13, 0), (13, 336))
98: ((13, 337), (13, 690))
230: ((2, 726), (2, 910))
70: ((14, 0), (14, 630))
109: ((15, 0), (15, 757))
178: ((4, 798), (4, 950))
239: ((0, 0), (0, 169))
201: ((0, 170), (0, 471))
1: ((0, 472), (0, 919))
254: ((0, 920), (0, 970))
2: ((1, 0), (1, 810))
196: ((1, 811), (1, 946))
8: ((2, 0), (2, 279))
207: ((2, 280), (2, 622))
25: ((2, 623), (2, 725))
31: ((3, 0), (3, 982))
42: ((4, 0), (4, 797))
178: ((4, 798), (4, 950))
165: ((5, 0), (5, 389))
35: ((5, 390), (5, 964))
200: ((6, 0), (6, 805))
20: ((7, 0), (7, 890))
13: ((8, 0), (8, 510))
145: ((8, 511), (8, 811))
16: ((9, 0), (9, 937))
212: ((10, 0), (10, 994))
57: ((11, 0), (11, 246))
111: ((11, 247), (11, 910))
127: ((12, 0), (12, 955))
132: ((13, 0), (13, 336))
98: ((13, 337), (13, 690))
70: ((14, 0), (14, 630))
109: ((15, 0), (15, 757))
19: ((16, 0), (16, 651))
172: ((2, 726), (2, 976))
205: ((6, 806), (6, 1015))
((2, 280), (2, 622))
239: ((0, 0), (0, 169))
201: ((0, 170), (0, 471))
1: ((0, 472), (0, 919))
254: ((0, 920), (0, 970))
2: ((1, 0), (1, 810))
196: ((1, 811), (1, 946))
8: ((2, 0), (2, 279))
207: ((2, 280), (2, 622))
25: ((2, 623), (2, 725))
172: ((2, 726), (2, 976))
31: ((3, 0), (3, 982))
178: ((4, 798), (4, 950))
165: ((5, 0), (5, 389))
35: ((5, 390), (5, 964))
200: ((6, 0), (6, 805))
205: ((6, 806), (6, 1015))
20: ((7, 0), (7, 890))
13: ((8, 0), (8, 510))
145: ((8, 511), (8, 811))
16: ((9, 0), (9, 937))
212: ((10, 0), (10, 994))
57: ((11, 0), (11, 246))
111: ((11, 247), (11, 910))
127: ((12, 0), (12, 955))
132: ((13, 0), (13, 336))
98: ((13, 337), (13, 690))
70: ((14, 0), (14, 630))
109: ((15, 0), (15, 757))
19: ((16, 0), (16, 651))
((14, 0), (14, 630))
239: ((0, 0), (0, 169))
201: ((0, 170), (0, 471))
1: ((0, 472), (0, 919))
254: ((0, 920), (0, 970))
2: ((1, 0), (1, 810))
196: ((1, 811), (1, 946))
8: ((2, 0), (2, 279))
207: ((2, 280), (2, 622))
25: ((2, 623), (2, 725))
172: ((2, 726), (2, 976))
31: ((3, 0), (3, 982))
178: ((4, 798), (4, 950))
165: ((5, 0), (5, 389))
35: ((5, 390), (5, 964))
200: ((6, 0), (6, 805))
205: ((6, 806), (6, 1015))
20: ((7, 0), (7, 890))
13: ((8, 0), (8, 510))
145: ((8, 511), (8, 811))
16: ((9, 0), (9, 937))
212: ((10, 0), (10, 994))
57: ((11, 0), (11, 246))
111: ((11, 247), (11, 910))
127: ((12, 0), (12, 955))
132: ((13, 0), (13, 336))
98: ((13, 337), (13, 690))
109: ((15, 0), (15, 757))
19: ((16, 0), (16, 651))
((9, 0), (9, 937))
239: ((0, 0), (0, 169))
201: ((0, 170), (0, 471))
1: ((0, 472), (0, 919))
254: ((0, 920), (0, 970))
196: ((1, 811), (1, 946))
8: ((2, 0), (2, 279))
207: ((2, 280), (2, 622))
25: ((2, 623), (2, 725))
172: ((2, 726), (2, 976))
31: ((3, 0), (3, 982))
178: ((4, 798), (4, 950))
165: ((5, 0), (5, 389))
35: ((5, 390), (5, 964))
200: ((6, 0), (6, 805))
205: ((6, 806), (6, 1015))
20: ((7, 0), (7, 890))
13: ((8, 0), (8, 510))
145: ((8, 511), (8, 811))
16: ((9, 0), (9, 937))
212: ((10, 0), (10, 994))
57: ((11, 0), (11, 246))
111: ((11, 247), (11, 910))
127: ((12, 0), (12, 955))
132: ((13, 0), (13, 336))
98: ((13, 337), (13, 690))
109: ((15, 0), (15, 757))
19: ((16, 0), (16, 651))
((5, 390), (5, 964))
((12, 0), (12, 955))
201: ((0, 170), (0, 471))
1: ((0, 472), (0, 919))
254: ((0, 920), (0, 970))
196: ((1, 811), (1, 946))
8: ((2, 0), (2, 279))
207: ((2, 280), (2, 622))
25: ((2, 623), (2, 725))
172: ((2, 726), (2, 976))
31: ((3, 0), (3, 982))
178: ((4, 798), (4, 950))
165: ((5, 0), (5, 389))
35: ((5, 390), (5, 964))
200: ((6, 0), (6, 805))
205: ((6, 806), (6, 1015))
20: ((7, 0), (7, 890))
13: ((8, 0), (8, 510))
145: ((8, 511), (8, 811))
16: ((9, 0), (9, 937))
212: ((10, 0), (10, 994))
57: ((11, 0), (11, 246))
111: ((11, 247), (11, 910))
127: ((12, 0), (12, 955))
132: ((13, 0), (13, 336))
98: ((13, 337), (13, 690))
109: ((15, 0), (15, 757))
19: ((16, 0), (16, 651))
((10, 0), (10, 994))
112: ((1, 0), (1, 533))
45: ((14, 0), (14, 1017))
202: ((4, 0), (4, 574))
((2, 726), (2, 976))
((13, 337), (13, 690))
161: ((1, 534), (1, 710))
58: ((0, 0), (0, 6))
58: ((0, 0), (0, 6))
201: ((0, 170), (0, 471))
1: ((0, 472), (0, 919))
254: ((0, 920), (0, 970))
112: ((1, 0), (1, 533))
161: ((1, 534), (1, 710))
196: ((1, 811), (1, 946))
8: ((2, 0), (2, 279))
207: ((2, 280), (2, 622))
25: ((2, 623), (2, 725))
172: ((2, 726), (2, 976))
31: ((3, 0), (3, 982))
202: ((4, 0), (4, 574))
178: ((4, 798), (4, 950))
165: ((5, 0), (5, 389))
35: ((5, 390), (5, 964))
200: ((6, 0), (6, 805))
205: ((6, 806), (6, 1015))
20: ((7, 0), (7, 890))
13: ((8, 0), (8, 510))
145: ((8, 511), (8, 811))
16: ((9, 0), (9, 937))
57: ((11, 0), (11, 246))
111: ((11, 247), (11, 910))
127: ((12, 0), (12, 955))
132: ((13, 0), (13, 336))
98: ((13, 337), (13, 690))
45: ((14, 0), (14, 1017))
109: ((15, 0), (15, 757))
19: ((16, 0), (16, 651))
((11, 247), (11, 910))
14: ((10, 0), (10, 760))
105: ((17, 0), (17, 880))
3: ((18, 0), (18, 789))
245: ((4, 575), (4, 772))
255: ((0, 0), (0, 0))
135: ((19, 0), (19, 560))
155: ((20, 0), (20, 874))
151: ((21, 0), (21, 614))
226: ((10, 761), (10, 1023))
186: ((22, 0), (22, 551))
193: ((23, 0), (23, 688))
95: ((0, 7), (0, 130))
241: ((8, 812), (8, 983))
30: ((0, 0), (0, 0))
160: ((24, 0), (24, 1016))
93: ((25, 0), (25, 873))
199: ((26, 0), (26, 966))
238: ((13, 691), (13, 989))
34: ((22, 552), (22, 1018))
27: ((1, 711), (1, 764))
216: ((15, 758), (15, 925))
153: ((16, 652), (16, 792))
17: ((19, 561), (19, 979))
144: ((21, 615), (21, 898))
89: ((27, 0), (27, 774))
208: ((28, 0), (28, 722))
125: ((29, 0), (29, 715))
167: ((30, 0), (30, 927))
48: ((16, 793), (16, 943))
39: ((23, 689), (23, 959))
209: ((31, 0), (31, 444))
156: ((0, 0), (0, 0))
212: ((32, 0), (32, 875))
222: ((18, 790), (18, 968))
166: ((7, 891), (7, 980))
163: ((0, 971), (0, 1022))
36: ((31, 445), (31, 975))
85: ((33, 0), (33, 731))
2: ((34, 0), (34, 925))
217: ((35, 0), (35, 1000))
91: ((36, 0), (36, 611))
24: ((37, 0), (37, 577))
228: ((17, 881), (17, 1012))
